mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MailboxMessageToElasticSearchJsonTest.java

```diff
@@ -96,13 +99,15 @@ public void spamEmailShouldBeWellConvertedToJson() throws IOException {
         MessageToElasticSearchJson messageToElasticSearchJson = new MessageToElasticSearchJson(
             new DefaultTextExtractor(),
             ZoneId.of("Europe/Paris"));
-        MailboxMessage spamMail = new SimpleMailboxMessage(date,
-            SIZE,
-            BODY_START_OCTET,
-            new SharedByteArrayInputStream(IOUtils.toByteArray(ClassLoader.getSystemResourceAsStream("eml/spamMail.eml"))),
-            new Flags(),
-            propertyBuilder,
-            MAILBOX_ID);
+        MailboxMessage spamMail = new SimpleMailboxMessage(
+                new TestMessageId.Factory().generate(),
+                date,
+                SIZE,
+                BODY_START_OCTET,
+                new SharedByteArrayInputStream(IOUtils.toByteArray(ClassLoader.getSystemResourceAsStream("eml/spamMail.eml"))),
+                new Flags(),
+                propertyBuilder,
+                MAILBOX_ID);
         spamMail.setUid(UID);
         spamMail.setModSeq(MOD_SEQ);
         assertThatJson(messageToElasticSearchJson.convertToJson(spamMail, ImmutableList.of(new MockMailboxSession("username").getUser())))
```

Comment:
and then messageId is not in the json output ? And when we'll add it, tests will break ?